skyportal/handlers/api/internal/plot.py

```diff
@@ -184,3 +184,19 @@ def get(self, obj_id, telescope_id):
 
         self.verify_and_commit()
         return self.success(data=json)
+
+
+class FilterWavelengthHandler(BaseHandler):
```

Comment:
I added it to the photometry endpoint

Reply:
never mind, I don't think that would work unless I added a filter_wavelength column to the photometry model :( I'll try querying via the filter endpoint

Reply:
per the conversation at thursday's standup, will leave this as its own internal handler. I guess the thing I'm unsure of is using a post or get request - I'm using a post right now so I can use the body of the request to send the filters along. but it doesn't seem quite right because nothing is actually being "posted".

Reply:
I think you should be using GET here. The only thing you are sending is a list of filter names, right? You can easily send them via query arguments (if it is an array of filter names, make sure to use `get_query_arguments` plural).